protos/protos/public/modeldb/DatasetVersionService.proto

```diff
@@ -301,12 +283,6 @@ service DatasetVersionService {
         };
     };
 
-    rpc getDatasetVersionTags(GetTags) returns (GetTags.Response) {
-        option (google.api.http) = {
-            get: "/v1/dataset-version/getDatasetVersionTags"
```

Comment:
we are not supporting the get..Tag, since no one was using it. 
webapp was using get Hydrated calls but with the decision, of moving to graphql may be we can not remove it